lms/djangoapps/discussion/static/discussion/js/views/discussion_user_profile_view.js

```diff
@@ -48,7 +48,7 @@
                         return view;
                     });
                     pageUrlFunc = function(page) {
-                        baseUri.clone().addSearch('page', page);
+                        return baseUri.clone().addSearch('page', page).toString();
```

Comment:
Why is toString() needed here?  Is this an HTML Snippet or something else?  If it is an HTML Snippet (meaning it is already escaped), we might want this function to return the HTML Snippet.